R/tm_file_viewer.R

```diff
@@ -8,18 +8,21 @@
 #' @inheritParams shared_params
 #' @param input_path (`list`) of the input paths, optional. Each element can be:
 #'
-#' 1. Specific path to files of accepted formats
-#' 2. A directory or a URL
+#' @inherit shared_params return
 #'
-#' The paths can be specified as absolute paths or relative to the running
-#' directory of the application.
-#' Default to the current working directory if not supplied.
+#' @examples
```

Comment:
I think the `@examples` is twice in this file now. 
The `data<-` part should be moved lower, before the `app <-`. Now it is before the description

Reply:
:roll_eyes:

Reply:
Better now?

Reply:
@chlebowa I think you need to move `@examples` tag just before `data <- ` line, and should be good. Now `@examples` tag is before some text about `Paths can be specifiied... bla bla` which is a part of the description. Now it's better, but the tag should be moved lover.

Reply:
Oh, thats supposed to be in `Details`!

Reply:
I haven't seen this before :thinking: 
![image](https://github.com/insightsengineering/teal.modules.general/assets/114988527/2637dba4-3783-4ffb-ace9-62c77c357d9c)

Reply:
I think this was some remainings of resolving a merge conflict. All good. We caught that and tests would caught that on `pre-release@main`. Thanks for yout help